### ISSUE TITLE
harbor.internalTLS.exporter.secretName not defined

### ISSUE THREAD

**Issue by ErmakovDmitriy** [goharbor/harbor-helm#871]
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
I tried to deploy Harbor 2.2.0 with Helm and internalTLS enabled.
I got an error:
```
Error: UPGRADE FAILED: template: harbor/templates/exporter/exporter-dpl.yaml:74:34: executing "harbor/templates/exporter/exporter-dpl.yaml" at <{{template "harbor.internalTLS.exporter.secretName" .}}>: template "harbor.internalTLS.exporter.secretName" not defined
```
I expected correct generation of the Helm templates when:
- internalTLS is enabled
- metrics is enabled

**Steps to reproduce the problem:**
1. Get Helm Chart from goharbor.io
2. modify values.yaml to set: internalTLS.enabled=true; metrics.enabled=true
3. Try to install the Helm chart.
4. Get the error

**Versions:**
Please specify the versions of following systems.

- Helm Chart version:  [1.6.0]
- Harbor version: [2.2.0]

** Possible solution**
You could try to add a Template definition to _helpers.tpl:

```
{{- define "harbor.internalTLS.exporter.secretName" -}}
  {{- if eq .Values.internalTLS.certSource "secret" -}}
    {{- .Values.internalTLS.exporter.secretName -}}
  {{- else -}}
    {{- printf "%s-exporter-internal-tls" (include "harbor.fullname" .) -}}
  {{- end -}}
{{- end -}}
```

It fixed the issue for me.

**Comment by jsaulalonso** [comment]
Same issue here, fixed on #882

**Comment by ErmakovDmitriy** [comment]
Thank you!
